src/api/app/components/notification_component.rb

```diff
@@ -24,7 +24,7 @@ def initialize(notification:, selected_filter:, page:, show_more:, current_user:
   end
 
   def notification_icon
-    if @notification.event_type.in?(['Event::RelationshipCreate', 'Event::RelationshipDelete'])
+    if @notification.event_type.in?(['Event::RelationshipCreate', 'Event::RelationshipDelete', 'Event::AddedUserToGroup'])
```

Comment:
@saraycp my bad, I forgot to add the `Event::RemovedUserFromGroup` here

Reply:
No problem